src/RefractDataStructure.cc

```diff
@@ -1050,7 +1060,7 @@ namespace drafter {
         return element;
     }
 
-    refract::IElement* ExpandRefract(refract::IElement* element, const refract::Registry& registry)
+    refract::IElement* ExpandRefract(refract::IElement* element, ConversionContext& context, const refract::Registry& registry)
```

Comment:
We don't need the third argument here. We can get the registry from the context which is the second argument.